packages/core/docs/vibe-3-changelog.md

```diff
@@ -1,9 +1,148 @@
 # Vibe 3 Changelog
 
+## Components
+
+### AttentionBox
+
+- `componentClassName` -> `className` [codemod]
```

Comment:
I thought it could be useful both for use to track of which can be changed via codemods and also let the users know that this change is covered by a codemod as well

Reply:
Love all the structure change of this changelog
Maybe the headline should be more specific stating components **props**?